partcad-cli/src/partcad_cli/click/commands/adhoc/convert.py

```
@@ -0,0 +1,107 @@
+            "brep",
+            "stl",
+            "3mf",
+            "obj",
+        ],
+        case_sensitive=False,
+    ),
+)
+@click.argument("input_filename", type=click.Path())
+@click.argument("output_filename", type=click.Path(), required=False)
+@click.pass_context
+def cli(ctx, create_dirs, output_dir, input, output, input_filename, output_filename):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)